### PR TITLE
crackle: init at unstable-2020-12-13

### PR DESCRIPTION
**Pull request by moni-dz**
###### Motivation for this change
#81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 124172` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>crackle</li>
  </ul>
</details>

File: pkgs/tools/networking/crackle/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, libpcap, coreutils }:
+
+stdenv.mkDerivation {
+  pname = "crackle";
+  version = "unstable-2020-12-13";
```
Comment (fabaff) [inline pkgs/tools/networking/crackle/default.nix:5]: Let's see if can get a proper release: https://github.com/mikeryan/crackle/issues/50

File: pkgs/tools/networking/crackle/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchFromGitHub {
+    owner = "mikeryan";
+    repo = "crackle";
```
Comment (fabaff) [inline pkgs/tools/networking/crackle/default.nix:9]: Opinionated change

```suggestion
    repo = pname;
```

**Review by fabaff — COMMENTED** [review]
Looks good to me. No real blockers.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 124172` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>crackle</li>
  </ul>
</details>

File: pkgs/tools/networking/crackle/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/networking/crackle/default.nix:5]: No release was made so far.

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package built (see https://github.com/NixOS/nixpkgs/pull/124172#issuecomment-846614927)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-124172]$ ./results/crackle/bin/crackle 
Usage: crackle -i <input.pcap> [-o <output.pcap>] [-l <ltk>] [-s]
Cracks Bluetooth Low Energy encryption (AKA Bluetooth Smart)

Major modes:  Crack TK // Decrypt with LTK
[...]           
```

Looks good to me.

**Pull request merged**